cts/projection/elemMatch.json

```diff
@@ -0,0 +1,123 @@
+{
+  "number": {
+    "request": {
+      "find": "composites",
+      "filter": {},
+      "projection": {
+        "v": {
+          "$elemMatch": {
+            "$gte": {
+              "$numberInt": "42"
+            }
+          }
+        }
+      },
+      "sort": {
+        "_id": {
+          "$numberInt": "1"
+        }
+      },
+      "$db": "{{.Database}}"
+    },
+    "response": {
+      "cursor": {
+        "firstBatch": [
+          {
+            "_id": "array",
+            "v": [
+              {
+                "$numberInt": "42"
+              }
+            ]
+          },
+          {
+            "_id": "array-composite",
+            "v": [
+              {
+                "$numberDouble": "42.13"
+              }
+            ]
+          },
+          {
+            "_id": "array-composite-reverse",
+            "v": [
+              {
+                "$numberDecimal": "46.1"
+              }
+            ]
+          },
+          {
+            "_id": "array-document-empty"
+          },
+          {
+            "_id": "array-documents"
```

Comment:
Why does `array-documents` not have any `v`?

Reply:
Because of the projection in the request. (The array of documents doesn't contain value that is greater, or equal to 42)

Reply:
:man_facepalming: